UnityProject/Assets/Scripts/UI/ResponsiveUI.cs

```diff
@@ -109,22 +109,6 @@ private IEnumerator ForceGameWindowAspect()
 			yield return new WaitForSeconds(0.2f);
 			if (!Screen.fullScreen)
 			{
-				float screenWidth = Screen.height * targetAspect;
-
-				//The following conditions check if the screen width or height
-				//is an odd number. If it is, then it adjusted to be an even number
-				//This fixes the sprite bleeding between tiles:
-				if ((int) screenWidth % 2 != 0)
-				{
-					screenWidth += 1f;
-				}
-				int screenHeight = Screen.height;
-				if (screenHeight % 2 != 0)
-				{
-					screenHeight++;
-				}
```

Comment:
These lines that check the screenHeight and screenWidth are making sure that the Height and Width are even numbers because if the they end up being odd numbers then you see gaps in between the tiles in the game. Removing this will reintroduce that bug.

Reply:
Ah, alright, I did a small check and I can confirm that bleedthrough happens when the Screen size isn't divisible by two.
However, using `Screen.SetResolution(...)` isn't good either as it prevents the window from fully resizing.
A possible solution would be to use layout groups and padding in order to avoid bleedthrough and still enable fullscreen window, but I will have to read up on those topics.